website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+     </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Specifies whether the pixels in a deep image are sorted and
+         non-overlapping.
+       </p>
+
+       <p style="padding-bottom:15px">
+         Note: this attribute can be set by application code that writes a
+         file in order to tell applications that read the file whether the
+         pixel data must be cleaned up prior to image processing operations
+         such as flattening.  The OpenEXR library does not verify that the
+         attribute is consistent with the actual state of the pixels.
+         Application software may assume that the attribute is valid, as long
+         as the software will not crash or lock up if any pixels are
+         inconsistent with the <tt>deepImageState</tt> attribute.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> idManifest </b> </tt>
+     </td>
+     <td style="vertical-align: top; width:100px"> <tt> CompressedIDManifest
+     </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         ID manifest.
+       </p>
+     </td>
+   </tr>
+
+   </table>
+   </embed>
+
+Deprecated Attributes
+=====================
+
+These attributes are how obsolete and are no longer officially supported by file
+format. Note that you can still read and write images that contain these
+attributes.
+
+.. raw:: html
+
+   <embed>
+   <table>
+   
+   <tr>
+     <td style="vertical-align: top; width:150px; font-weight:bold"> attribute
+     name <rat/td>
+     <td style="vertical-align: top; width:500px; font-weight:bold"> definition </td>
+   </tr>
+
+   <tr style="border-width:3px">
+     <td style="vertical-align: top; width:150px"> <tt> <b> dwaCompressionLevel
+     </b> </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Sets the quality level for images compressed with the DWAA or DWAB method.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> renderingTransform, lookModTransform
+     </b> </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Specify the names of the CTL functions that implements the intended
+         color rendering and look modification transforms for this image.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> maxSamplesPerPixel
+     </b> </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Stores the maximum number of samples used by any single pixel within a
+         deep image. If this number is small, it may be appropriate to read the
+         deep image into a fix-sized buffer for processing. However, this number
+         may be very large.
```

Comment:
We should also add that the library never enforced the correctness of this value, so if it shows up in legacy files it should not be trusted.

Reply:
Done.